mriqc/qc/tests/test_anatomical.py

```diff
@@ -35,7 +34,7 @@
 from ..anatomical import art_qi2
 
 
-class GroundTruth(object):
```

Comment:
In Python 3, [`object`](https://docs.python.org/3/library/functions.html#object) inheritance is implicit.